### ISSUE TITLE
Cannot be compiled with GHC 8.0.1

### ISSUE THREAD

**Issue by Qinka** [MarcWeber/hasktags#23]
There is an error:
```
[1 of 3] Compiling DebugShow        ( src/DebugShow.hs, dist/build/DebugShow.o )

src/DebugShow.hs:17:1: warning: [-Wredundant-constraints]
    • Redundant constraint: Show a
    • In the type signature for:
           trace_ :: Show a => String -> a -> b -> b
[2 of 3] Compiling Tags             ( src/Tags.hs, dist/build/Tags.o )

src/Tags.hs:19:1: warning: [-Wtabs]
    Tab character found here, and in 38 further locations.
    Please use spaces instead.
[3 of 3] Compiling Hasktags         ( src/Hasktags.hs, dist/build/Hasktags.o )

src/Hasktags.hs:495:5: error:
    Ambiguous occurrence ‘isSymbolicLink’
    It could refer to either ‘System.Directory.isSymbolicLink’,
                             imported from ‘System.Directory’ at src/Hasktags.hs:31:1-23
                          or ‘System.Posix.Files.isSymbolicLink’,
                             imported from ‘System.Posix.Files’ at src/Hasktags.hs:38:1-25
                             (and originally defined in ‘unix-2.7.2.0:System.Posix.Files.Common’)
cabal: Leaving directory '/var/folders/bw/5sdnmnqj49z3my331n43w3hc0000gn/T/cabal-tmp-599/hasktags-0.69.1'
cabal: Error: some packages failed to install:
hasktags-0.69.1 failed during the building phase. The exception was:
ExitFailure 1
```

With Cabal 1.24 and GHC 8.0.1

**Comment by mchaver** [comment]
I ran into the same issue with a slightly earlier version of GHC. My quick solution was just to download the package and alter line 495 of `src/Hasktags.hs` to

```haskell
    System.Posix.Files.isSymbolicLink `fmap` getSymbolicLinkStatus p
```

**Comment by jhenahan** [comment]
Fixed by #30, will be in the next release shortly.
